core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -82,6 +85,17 @@ class GroupMetadataManager(brokerId: Int,
    * We use this structure to quickly find the groups which need to be updated by the commit/abort marker. */
   private val openGroupsForProducer = mutable.HashMap[Long, mutable.Set[String]]()
 
+  /* setup metrics*/
+  private val metricConfig: MetricConfig = new MetricConfig().samples(1)
```

Comment:
we want to avoid "bare numbers" like that. Can you use a final static variable instead? with meaningful name? It is hard to know what 1 means here (one what?) and what is it significance (what will happen if I use 2?)

Reply:
Hmm I skipped over this, but it seems like we don't get much benefit from having an both an average and a max if we have only keep one sample. Perhaps we could just leave this at the default of 2?

Reply:
Changed to default; In order to remove "bare numbers" and since it's not much different with 2 samples or 1 sample, it's best to stick with the default of 2 samples. 
The max will instead be taken over 2 samples of 30sec rather than 1 sample of 30sec.